src/hubbardoperators.jl

```diff
@@ -3,19 +3,27 @@ module HubbardOperators
 using TensorKit
 
 export hubbard_space
+export S_x, S_y, S_z, S_plus, S_min
 export e_plus_e_min, u_plus_u_min, d_plus_d_min
 export e_min_e_plus, u_min_u_plus, d_min_d_plus
+export u_min_d_min, d_min_u_min
 export e_num, u_num, d_num, ud_num
-export e_hopping
+export e_hopping, singlet_min
+export S_plus_S_min, S_min_S_plus, S_exchange
 
+export Sˣ, Sʸ, Sᶻ, S⁺, S⁻
 export e⁺e⁻, u⁺u⁻, d⁺d⁻, e⁻e⁺, u⁻u⁺, d⁻d⁺
 export n, nꜛ, nꜜ, nꜛꜜ
-export e_hop
+export e_hop, singlet⁻
+export S⁻S⁺, S⁺S⁻
 
 """
     hubbard_space(particle_symmetry::Type{<:Sector}, spin_symmetry::Type{<:Sector})
 
-Return the local hilbert space for a Hubbard-type model with the given particle and spin symmetries.
+Return the local hilbert space for a Hubbard-type model with the given particle and spin symmetries. The four basis states are
+```
+    |0⟩ (vacuum), |↑⟩ = (c↑)†|0⟩, |↓⟩ = (c↓)†|0⟩, |2⟩ = (c↑)†(c↓)†|0⟩.
```

Comment:
```suggestion
    |0⟩ (vacuum), |↑⟩ = (c↑)†|0⟩, |↓⟩ = (c↓)†|0⟩, |↑↓⟩ = (c↑)†(c↓)†|0⟩.
```
I like this slightly more since it explicitly adds the order of the doubly occupied state, but I'm okay with both

Reply:
I chose |2⟩ because I thought |↑↓⟩ might be confused with |↑_1, ↓_2⟩. So I prefer |2⟩ as long as we make it clear about the internal order.

Reply:
I guess for two sites we add a comma, but I see your point. In general we haven’t been very consistent with the Unicode anyways I think. I also just discovered there is ⇅ as well.

Reply:
Now that I'm in the minority, I just changed the notation to `↑↓`. Different sites are separated by commas.